MicrosoftDX/nether
# Analytics Deployment

### Issue: N/A


### Description:
Enhancements to bring the Event Hub into the "master" deployment. Includes integration with the environment settings for the web application. Also contains an initial and untested draft of the analytics template.  

### Test:
1) Run the deployment per the instructions on the in deployment/readme using the deploy.ps1 powershell script. 
2) Deploy the schema to the created "nether" database
3) Go to the https swagger endpoint and very you can successfully authorize using the swagger admin user and password
*if step 3 passes*  
4) run the integration tests against the deployed site 
5) Using service bus explorer or the Azure portal, ensure that messages have been sent to the event hub

Note: if using an existing resource group, remove the web site app service resource prior to running the deployment to ensure that all environment variables are "prestine". You can also skip step 2 if your nether database already has the base schema. 

(Make sure that all above information is provided before submitting your pull request)
